browser-only.js

```diff
@@ -15,6 +15,8 @@ require('webrtc-adapter');
 
 require('whatwg-fetch');
 
+require('function.prototype.name').shim();
```

Comment:
i feel like maybe this would be better in js-shims, but in an IE 9-11 target. Let's chat about it tomorrow.